### PR TITLE
Publish Anonymizer and Analyzer to PyPI

### PR DESCRIPTION
**Pull request by SharonHart**
Used the legacy pipeline template as reference:
https://github.com/microsoft/presidio/blob/master/pipelines/templates/publish-python-twine.yaml
Mainly, removed ADO artifacts publishing, which simplified the pipeline by removing conditions and most of the parameters.

Pass analyzer and anonymizer working dirs as steps arguments

### CODE REVIEW

File: .pipelines/templates/release.yml [hunk 1]
```diff
@@ -27,6 +28,17 @@ jobs:
         tagSource: manual
         tag: $(version)
 
+- job: PublishToPyPi
+  displayName: Publish to PyPi
+  dependsOn: version
+  steps:
+    - template: ./publish-to-pypi.yml
```
Comment (balteravishay) [inline .pipelines/templates/release.yml:35]: can be a matrix and then run in paralllel.

File: .pipelines/templates/publish-to-pypi.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    inputs:
+      pythonUploadServiceConnection: Pypi-Presidio
+  - task: Bash@3
+    displayName: 'Upload Wheel to PyPi'
+    inputs:
+      targetType: 'inline'
+      script: |
+        set -eux  # fail on error
+        echo ${PYPIRC_PATH}
+        ls -l
```
Comment (balteravishay) [inline .pipelines/templates/publish-to-pypi.yml:35]: ls -l: intentional?

**Review by balteravishay — APPROVED** [review]
minor comments

File: .pipelines/templates/publish-to-pypi.yml (hunk 2 above)
Comment (SharonHart) [reply .pipelines/templates/publish-to-pypi.yml:35]: no, so is the echo, good catch

**Comment by SharonHart** [comment]
@balteravishay Done, thanks

**Review by balteravishay — APPROVED** [review]

**Review by navalev — APPROVED** [review]
LGTM

**Pull request merged**